app/controllers/admin/tutorials_controller.rb

```diff
@@ -21,6 +31,10 @@ def update
 
   private
 
+  def new_tutorial_params
+    params.require(:tutorial).permit(:id, :title, :description, :thumbnail, :playlist_id)
+  end
+
```

Comment:
Do we need to still permit :playlist_id if it is removed from the form? Also, should we leave it as an optional field in the form? I'm not sure when a playlist_id could be added to a tutorial if we don't have it in the create form at all.